src/main/docker/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM frolvlad/alpine-oraclejdk8:slim
```

Comment:
I'd prefer using official images. For example `openjdk:8u131-jre-alpine`.
For runtime we need JRE, not JDK.
Locking specific dependency version is important for build reproducibility.